sdks/python/apache_beam/typehints/schemas.py

```diff
@@ -463,19 +463,20 @@ def named_tuple_from_schema(self, schema: schema_pb2.Schema) -> type:
 
     # Define a reduce function, otherwise these types can't be pickled
     # (See BEAM-9574)
-    def __reduce__(self):
-      return (
-          _hydrate_namedtuple_instance,
-          (schema.SerializeToString(), tuple(self)))
-
-    setattr(user_type, '__reduce__', __reduce__)
+    setattr(user_type, '__reduce__', _named_tuple_reduce)
```

Comment:
To clarify, Cloud Pickle only looks for a "__reduce__" method and setting the "__reduce__" attribute does not trigger proto pickling ?